README.md

```diff
@@ -2,70 +2,98 @@
    </a>
-
 </div>
 
+<h1 align="center">
+   <p>Open Source Unstructured API</p>
+</h1>
 
----
-
-<h3 align="center">
-  <p>General Pre-Processing Pipeline for Documents</p>
-</h3>
-
-This repo implements a pre-processing pipeline for the following documents. Currently, the pipeline is capable of recognizing the file type and choosing the relevant partition function to process the file.
+This repo implements a FastAPI server with the partitioning functionality of the [Unstructured library](https://github.com/Unstructured-IO/unstructured). It has one endpoint that accepts any of the following filetypes, and returns the contents as structured JSON.
 
 
 | Category  | Document Types                |
 |-----------|-------------------------------|
-| Plaintext | `.txt`, `.eml`, `.msg`, `.xml`, `.html`, `.md`, `.rst`, `.json`, `.rtf` |
-| Images    | `.jpeg`, `.png`               |
-| Documents | `.doc`, `.docx`, `.ppt`, `.pptx`, `.pdf`, `.odt`, `.epub`, `.csv`, `.tsv`, `.xlsx` |
+| Plaintext | `.csv`, `.eml`, `.html`, `.md`, `.msg`, `.rst`, `.rtf`, `.tsv`, `.txt`, `.xml`|
+| Images    | `.bmp`, `.heic`, `.jpeg`, `.png`, `.tiff`                            |
+| Documents | `.doc`, `.docx`, `.epub`, `.odt`, `.pdf`, `.ppt`, `.pptx`, `.xlsx` |
 | Zipped    | `.gz`                         |
 
+Any questions? Get in touch through our [Slack community](https://short.unstructured.io/pzw05l7)!
 
-## :rocket: Unstructured API
+## Quickstart
 
-Try our hosted API! It's freely available to use with any of the filetypes listed above. This is the easiest way to get started. If you'd like to host your own version of the API, jump down to the [Developer Quickstart Guide](#developer-quick-start).
+You can try it out by running the docker container and sending one of the sample files:
```

Comment:
"docker" -> "Docker"?

Also, would readers already know how to run the Docker container here? If not, consider linking to how to do it?